modules/gdscript/gdscript_parser.cpp

```diff
@@ -1799,24 +1799,27 @@ GDScriptParser::AssertNode *GDScriptParser::parse_assert() {
 	// TODO: Add assert message.
 	AssertNode *assert = alloc_node<AssertNode>();
 
+	push_multiline(true);
 	consume(GDScriptTokenizer::Token::PARENTHESIS_OPEN, R"(Expected "(" after "assert".)");
+
 	assert->condition = parse_expression(false);
 	if (assert->condition == nullptr) {
 		push_error("Expected expression to assert.");
 		complete_extents(assert);
 		return nullptr;
```

Comment:
It needs a `pop_multiline()` before this `return`, otherwise the multiline stack will have an extra element.

Reply:
Done.